prac2-pages/Trisha/login_page.dart

```diff
@@ -0,0 +1,136 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class LoginPage extends StatefulWidget {
+  const LoginPage({Key? key}) : super(key: key);
+  @override
+  State<LoginPage> createState() => _MyLoginState();
+  
+}
+
+class _MyLoginState extends State<LoginPage> {
+  bool valuefirst = false;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Color.fromARGB(255, 183, 173, 197),

```

Comment:
It is preferable to follow the design file, and so please remove the background color :thinking: .